modules/virt-configuring-ssh.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-ssh_{context}"]
+= Configuring SSH for quick virtual machine access
+
+You can configure your local SSH client to quickly connect to your virtual machine (VM). This configuration makes it possible to leverage `virtctl` to relay an SSH command to the target VM.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `virtctl` client.
+* The virtual machine you want to access is running.
+
+.Procedure
+
+. Modify the `~/.ssh/config` file on your client machine to include the following text:
++
+[source,terminal]
+----
+Host vmi/*

```

Comment:
@phoracek would be `vm/*` here more intuitive to the user? `vmi/*` and `vm/*` seems to work the same way, I do not understand the difference.
Or should there be even two entries, one for `vmi/*` and another one for `vm/*` ?

Reply:
`vm/*` and `vmi/*` work the same in case of `virtctl port-forward`. But Dominik is right, we should make `vm/*` the preferred method.